sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/InstancesSamples.cs

```diff
@@ -22,14 +22,13 @@ public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
             PrintHeader("TIME SERIES INSIGHTS INSTANCES SAMPLE");
 
             // Figure out what keys make up the Time Series Id
-            TimeSeriesModelSettings modelSettings = await client.ModelSettings.GetAsync().ConfigureAwait(false);
+            TimeSeriesModelSettings modelSettings = await client.ModelSettings.GetAsync();
 
-            TimeSeriesId tsId = TimeSeriesIdHelper.CreateTimeSeriesId(modelSettings);
+            #region Snippet:TimeSeriesInsightsSampleCreateInstance
 
+            TimeSeriesId tsId = TimeSeriesIdHelper.CreateTimeSeriesId(modelSettings);
```

Comment:
So it seems like the snippet now references TimeSeriesIdHelper. So when this is presented on msdocs, I am afraid developers will think this is something we ship with the NuGet package. What are your thoughts on leaving this specific line out of the snippet context, and just mention in a comment that tsId has already been created earlier? We probably don't even need to mention that tsid has been created above. I am OK with either.